_events/2016-06-11.md

```diff
@@ -0,0 +1,39 @@
+---
+#published: false # Remove this line for real events.
+title: "New Musical Instruments Hackathon"
+description: "A Monthly Music Hackathon Series installation with  theme 'new musical instruments'."
+date: 2016-06-11 11:30am
+address: "45 W 18th St, New York, NY 10011"
+rsvp: "https://www.eventbrite.com/o/monthly-music-hackathon-nyc-2470452960"
+image: "2016-06-11.gif"
+---
+
+# Schedule
+
+- 11:30 AM Doors open
+- 12:00 PM Talks by special guests, and lunch
+- 1:30 PM Hacking, brainstorming session, and workshops start
+- 8:00 PM Performance of a section of Jay Allen Zimmerman’s Roboticus
+- 8:15 PM Demo / Performance of EMMI’s Cytherbot.
+- 8:25 PM Demos of hacks
+
+# Talks
+
+- Thomas Carroll
+- Lori Napoleon
+- Scott Barton & Steven Kemper
+- Jessica Segal
+- Matthew Ostrowski
+
+# Workshops
+
+- Jay Allen Zimmerman, Roboticus
+
+# Details
+The [Monthly Music Hackathon]({{site.baseurl}}) series presents our [New Musical Instruments Hackathon]({{site.baseurl}}{{link _events/2016-06-11.md}} "New Musical Instruments Hackathon 2016"):  a day-long, non-competitive, creative event exploring new ways to make music, noise, & sound.  From musical robots to hyperinstruments to making wind instruments out of pipes, we’ve got it all. The event is open to hackers and aspiring hackers of all skill levels and disciplines, utilizing technology or not.
+
+Our presenters include [Thomas Carroll](http://www.carrollclarinet.com/ "Thomas Carroll"), who builds clarinets and historical wind instruments; [Lori Napoleon (Merdian7, Antenes)](http://meridian7.net/ "Lori Napoleon"), who builds modular synths from old telephones switchboards; Scott Barton & Steven Kemper, who make musical robots; [Jessica Segal](http://www.jessicasegall.com/ "Jessica Segal"), who makes music from bees and old pianos; and Matthew Ostrowski, who makes music out of rotary phones.
+
+We’re also planning workshops on new instrument hardware, interactive dance & gestural music. Performance-wise, we’ll have an excerpt of [Jay Allen Zimmerman](http://www.musicbyjaz.com "Jay Allen Zimmerman")’s Roboticus, along with music from the EMMI bots.
```

Comment:
In general, I don't think we need to write the link title/hover text.
I mean, just more effort for not much gained. I dunno; what do you think?